dlt/common/storages/exceptions.py

```diff
@@ -68,31 +69,30 @@ def __init__(self, load_id: str, supported_formats: Iterable[str], wrong_job: st
         self.expected_file_formats = supported_formats
         self.wrong_job = wrong_job
         super().__init__(
-            f"Job {wrong_job} for load id {load_id} requires job file format that is not one of"
-            f" {supported_formats}"
+            f"Job `{wrong_job}` for load id `{load_id}` requires job file format that is not one"
+            f" of: {supported_formats}"
         )
 
 
 class LoadPackageNotFound(LoadStorageException, FileNotFoundError):
     def __init__(self, load_id: str) -> None:
         self.load_id = load_id
-        super().__init__(f"Package with load id {load_id} could not be found")
+        super().__init__(f"Package with `{load_id=:}` could not be found")
 
 
 class LoadPackageAlreadyCompleted(LoadStorageException):
     def __init__(self, load_id: str) -> None:
         self.load_id = load_id
         super().__init__(
-            f"Package with load id {load_id} is already completed, but another complete was"
-            " requested"
+            f"Package with `{load_id=:}` is already completed, but another complete was requested"
         )
 
 
 class LoadPackageNotCompleted(LoadStorageException):
     def __init__(self, load_id: str) -> None:
         self.load_id = load_id
         super().__init__(
-            f"Package with load id {load_id} is not yet completed, but method required that"
+            f"Package with `{load_id=:}` is not yet completed, but method required that"
```

Comment:
nitpick: `but method required that` reads like its missing what method. maybe this could be `but the calling method required it to be`